.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,8 +29,8 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 - [ ] I have performed a self-review of my own code
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
-- [ ] My code follows the style guidelines of this project (I have run `yarn format`)
+- [ ] My code follows the style guidelines of this project (I have run `pnpm run prettier` and `pnpm run lint`)
 - [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] New and existing unit tests pass locally with my changes (I have run `yarn test`)
+- [ ] New and existing unit tests pass locally with my changes (I have run `pnpm run types:check`, `pnpm run test` and `pnpm run check:ci`)
```

Comment:
`pnpm run check:ci` should actually run the `types:check` and `test` scripts! I wonder if it'd make sense to condense these bullet points down to basically "I ran `pnpm run check:ci`..." since that command does all of the CI checks (formatting, linting, unit tests).

Reply:
Yeah I think condensing it makes sense!